modules/imap/site.css

```diff
@@ -138,14 +138,18 @@
   background-color: inherit;
   color: inherit;
 }
-.header_links #dropdownMenuForward {
+.header_links #dropdownMenuForward, .header_links #dropdownMenuSnooze, .header_links #dropdownMenuTag, .header_links #dropdownMenuNexterDate {
   padding: 0;
   border: unset;
   font-variant: inherit;
   text-transform: inherit;
   font-size: inherit;
   vertical-align: baseline;
 }
+.header_links .dropdown .btn:hover, .header_links .dropdown .btn:active, .header_links .dropdown .btn:focus {
+    color: inherit !important;
```

Comment:
Can you please avoid using important here in the stock css themes? There would be no way to override for the user afterwards.